apollo-adminservice/src/test/java/com/ctrip/framework/apollo/adminservice/aop/NamespaceLockTest.java

```diff
@@ -49,16 +47,11 @@
   private static final String ANOTHER_USER = "user-test2";
   private static final long NAMESPACE_ID = 100;
 
-  @Mock
-  private NamespaceLockService namespaceLockService;
-  @Mock
-  private NamespaceService namespaceService;
-  @Mock
-  private ItemService itemService;
-  @Mock
-  private BizConfig bizConfig;
-  @InjectMocks
-  NamespaceAcquireLockAspect namespaceLockAspect;
+  @Mock private NamespaceLockService namespaceLockService;
+  @Mock private NamespaceService namespaceService;
+  @Mock private ItemService itemService;
+  @Mock private BizConfig bizConfig;
+  @InjectMocks NamespaceAcquireLockAspect namespaceLockAspect;
```

Comment:
Is it possible to keep the annotations on top of the fields?

Reply:
Find the issue https://github.com/google/google-java-format/issues/648 about it.

It seem that this feature didn't support yet.

Maybe we can wait the feature.

Reply:
emm...I think this is an important feature as most of use would be more comfortable with the annotation in its own line?